src/registrar/assets/js/get-gov.js

```diff
@@ -1834,6 +1834,39 @@ document.addEventListener('DOMContentLoaded', function() {
         
         // When the edit button is clicked, show the input field under it
         handleEditButtonClick(fieldName, button);
+
+        let editableFormGroup = button.parentElement.parentElement.parentElement;
```

Comment:
Seems using an ID or class would be better...is there an inconsistency in the DOM that prevents a more proper lookup?

Reply:
Seems like a red flag architecture-wise, but I see the reasoning.  Sounds good